book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,234 @@
+-   Conversely it must be physically stolen to be accessed by a malicious third party.
+-   On the other hand, ELNs - like all software - have security vulnerabilities.
+-   You are not dependent on the functioning of any complex systems like computer networks in order to be able to use your paper lab notebook.
+-   You do not need any specialist tools in order to access its contents.
+-   There is no reason why you might have to pay a 3rd party a fee in order to continue using it.
+-   You do not have to agree to a 'terms of service' or 'end-user license agreement' with a third party, the terms of which may be subject to unilateral alteration by that third party, in order to use and retain access to your lab book.
+
+If your provider of paper lab books goes out of business it has almost no bearing on your ability to continue doing your work.
+One paper notebook is much like another, finding a new provider is pretty easy.
+Changing providers does not impact on your ability to access your past notebooks or to continue operating with the same workflow in future ones.
+This is not necessarily true of ELNs.
+
+(rr-rdm-elns-choosing-archival)=
+### Archival function of notebooks
+
+Few active measures are needed to maintain the data in your paper notebooks, they are vulnerable as they exist in only one copy. 
+Nevertheless, as long as they are kept in a cool, dry and dark spot for example a fireproof safe they will likely last decades.
+Electronic data requires much more active upkeep.
+Electronic data can *if manged properly* be more resilient to physical threats such as fire and flooding as it can frequently be backed up in multiple locations.
+A hard drive or even a solid state drive however cannot be left in a draw for a decade or more and have a high likelihood of working without some amount of bitrot or compatability issue when plugged back in.
+
+Lab notebooks perform an archival function and proprietary formats are antithetical to this as they assume the institution which can act as a gatekeeper to the use of the proprietary format will outlive the need to archive the material.
+When choosing an archival format one seeks to maximize the likelihood that one can recover the relevant information from that format.
+Using a proprietary solution for archival purposes is talking a needless risk with the future of your data.
+Your data's fate can become tied to that of the company, or project, that develops and operates the software that you use to store your lab notes.
+
+(rr-rdm-elns-choosing-lockin)=
+### Lock-in
+
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it's the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
+The ability to export your data and retain it's structure is very important to evaluate.
```

Comment:
```suggestion
The ability to export your data and retain the original structure is very important to evaluate.
```

Reply:
Addressed in [e1318f5](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/e1318f5c6726d59c408b51fd05bf95b114e846b0)